chrobalt/BUILD.gn

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+group("all_chrobalt") {
```

Comment:
nit: IIRC it's more customary to call this "gn_all".

Reply:
That's correct, and it also seem unnecessary to have intermediate group like this which has a single dependency—add the starboard dependency directly here

Reply:
done